pkg/kubelet/winstats/winstats_test.go

```diff
@@ -123,6 +130,21 @@ func TestWinVersionInfo(t *testing.T) {
 		KernelVersion: "v42"})
 }
 
+func TestConvertCPUValue(t *testing.T) {
+	var cpuValue uint64 = 10000
+	var cpuCores = 4
+	p := perfCounterNodeStatsClient{}
+	newValue := p.convertCPUValue(cpuCores, cpuValue)
+	assert.Equal(t, newValue, uint64(400000000000))
+}
+
+func TestGetCPUUsageNanoCores(t *testing.T) {
+	p := perfCounterNodeStatsClient{}
+	p.cpuUsageCache = cpuUsageCache{200000000000, 500000000000}
+	cpuUsageNanoCores := p.getCPUUsageNanoCores()
+	assert.Equal(t, cpuUsageNanoCores, uint64(1229782918))
```

Comment:
Based on (500... - 200...) / 10..., shouldn't we expect a 3... for the result?

Reply:
I was taking the value returned from the expected value and thanks for noticing this! I fixed the test data for `cpuUsageCache` to use `uint64` type as well as fixed the calculation and the expected value is matching with real calculation now.